static/js/show_copied_confirmation.js

```diff
@@ -0,0 +1,20 @@
+import tippy from "tippy.js";
+
+export function show_copied_confirmation($copy_button) {
+    // Display a tooltip to notify the user the version was copied.
+    const instance = tippy($copy_button, {
+        placement: "top",
+        appendTo: () => document.body,
+        onUntrigger() {
+            remove_instance();
+        },
+    });
+    instance.setContent("Copied!");
+    instance.show();
+    function remove_instance() {
+        if (!instance.state.isDestroyed) {
+            instance.destroy();
+        }
+    }
+    setTimeout(remove_instance, 3000);
+}
```

Comment:
You don't need to create a new file for this, this can just exist in `tippyjs.js`.

Reply:
Ok, I will update it.